lib/error.js

```diff
@@ -13,10 +13,20 @@ function FormError(key, options, req, res) {
             return options.message || this.getMessage(key, options, req, res);
         }
     });
+    Object.defineProperty(this, 'title', {
+        enumerable: true,
+        get: function () {
+            return options.title || this.getTitle();
```

Comment:
Shouldn't the getTitle method also take the same arguments as getMessage.
(Assuming we are sticking with this pattern for now.)